### PR TITLE
(new package) Php

### PR DESCRIPTION
**Pull request by majkinetor**
Note: Due to vcredist dependency testing requries ~700MB downloads.

### CODE REVIEW

File: automatic/php/php.nuspec [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>php</id>
+    <title>PHP (Hypertext Preprocessor)</title>
+    <version>0.0</version>
+    <authors>PHP Authors</authors>
+    <owners>Rob Reynolds</owners>
```
Comment (AdmiringWorm) [inline automatic/php/php.nuspec]: missing chocolatey in the owners elements.

File: automatic/php/tools/chocolateyUninstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿$packageName = 'php'
+
+$installLocation = Join-Path $(Get-ToolsPath) $packageName
+Write-Host "Deleting $installLocation"
+rm -Force -Recurse $installLocation
+
+#Uninstall-ChocolateyPath $installLocation
```
Comment (AdmiringWorm) [inline automatic/php/tools/chocolateyUninstall.ps1]: why is this commented out?
shouldn't the install location be removed from path environment variable?

**Comment by AdmiringWorm** [comment]
I'm also getting an error when trying to uninstall this package:
```
 ERROR: The term 'Get-ToolsPath' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

File: automatic/php/tools/chocolateyUninstall.ps1 (hunk 2 above)
Comment (majkinetor) [inline automatic/php/tools/chocolateyUninstall.ps1]: AFAIK such function doesn't exist yet.

**Comment by majkinetor** [comment]
Thx worm, fixed.

File: automatic/php/tools/chocolateyUninstall.ps1 (hunk 2 above)
Comment (AdmiringWorm) [inline automatic/php/tools/chocolateyUninstall.ps1]: ah, I see. That sucks.
Guess I should have checked the function and variable docs first.

**Comment by ferventcoder** [comment]
Missing https://github.com/ferventcoder/chocolatey-packages/pull/195

**Comment by ferventcoder** [comment]
That addresses https://github.com/ferventcoder/chocolatey-packages/issues/145

**Comment by ferventcoder** [comment]
One request I have. All issues and PRs must be addressed for packages that are being pulled over.  https://github.com/ferventcoder/chocolatey-packages/search?q=PHP&state=open&type=Issues&utf8=%E2%9C%93 

Please make sure you are addressing the issues from the original repo. Yes, it's my bad that some of these issues have been around for awhile, but I'd rather get those things addressed so they are not lost as we convert things over.

**Comment by majkinetor** [comment]
If there are no other requrests I will merge this.

**Comment by gep13** [comment]
The chocolatey owner still hasn't been added to this package, so don't think we should merge this yet.  I will merge once I add the user to the package.

**Comment by gep13** [comment]
chocolatey user has been added to this package.

Pull request created here to remove php from @ferventcoder repo, as this didn't look like it was done.

**Comment by gep13** [comment]
@majkinetor I am going to go ahead and merge this, but I think you are still discussing an open issue on @ferventcoder repo, right?  Perhaps we should move that issue here, and close the one over there, that way it is easier to see that there is still some work outstanding.

**Comment by majkinetor** [comment]
> @majkinetor I am going to go ahead and merge this, but I think you are still discussing an open issue on @ferventcoder repo, right?

Kinda, but I think its OK as it is now.

> Pull request created here to remove php from @ferventcoder repo, as this didn't look like it was done.

I didn't do so as he created issue here. I do so for packages he didn't initiate such as vlc. I will be moving many of @ferventcoder packages here in the future. He has a list of WIP packages that are largely unmaintained.

**Comment by gep13** [comment]
>@majkinetor said...
Kinda, but I think its OK as it is now.

A quick search of that repo turned up these:
https://github.com/ferventcoder/chocolatey-packages/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20php

Which I think we should address in some way or another since we are assuming ownership of this package.

>@majkinetor said...
I didn't do so as he created issue here. I do so for packages he didn't initiate such as vlc. I will be moving many of @ferventcoder packages here in the future.

If we can work through the process that I outlined on the wiki for any packages that we are taking over, I think that will help with this transition phase.  I have commit rights into that repo so I can merge the PR's as/when required.

>@majkinetor said...
He has a list of WIP packages that are largely unmaintained.

Which comes from the fact that he is actively working on core chocolatey 😄 He has given a green light to moving his packages over to this repo though.

**Comment by AdmiringWorm** [comment]
> Which comes from the fact that he is actively working on core chocolatey 😄 He has given a green light to moving his packages over to this repo though.

All packages?

**Comment by gep13** [comment]
@AdmiringWorm that is my understanding, but I would direct you to the first bullet point here:

https://github.com/chocolatey/chocolatey-coreteampackages/wiki/package-migration-process

😄

**Comment by majkinetor** [comment]
> All packages?

I would move WIP ketarin packages, there at least 20 there (sysinternals, vagrant, putty ...).

**Comment by gep13** [comment]
@majkinetor what are you referring to exactly when you say WIP?  I realise this is Work In Progress, but is there something that highlights these packages?

**Comment by majkinetor** [comment]
Oh, its not WIP its VIP, Very important packages :) Sorry.

**Comment by ferventcoder** [comment]
Definitely the VIP packages

**Pull request merged**